pumps/sql_test.go

```diff
@@ -119,3 +120,161 @@ func TestSQLWriteDataSharded(t *testing.T) {
 	assert.Len(t, dbRecords, 2)
 
 }
+
+func TestSQLWriteUptimeData(t *testing.T) {
+	pmp := SQLPump{IsUptime: true}
+	cfg := make(map[string]interface{})
+	cfg["type"] = "sqlite"
+	cfg["connection_string"] = "pmp_test.db"
```

Comment:
maybe we can try in-mem to be consistent with other tests.

Reply:
Done